### DISCUSSION TITLE
UHGGv2 + UHGV MQ+ database with library fna file

### DISCUSSION THREAD

**Discussion by alienzj** [bhattlab/phanta#42] [Phanta DBs]
Hi, Phanta team member,

I downloaded UHGGV2plusUHGVmq database from [UHGGv2 + UHGV "MQ+"](http://ab_phanta.os.scg.stanford.edu/Phanta_DBs/uhgg2_uhgv_v1.tar.gz), but I can't find the library fna file.

Recently I want to build a custom phanta database using genomes of UHGGV2plusUHGVmq,  1586 fungi genomes downloaded from NCBI genbank, and other MAGs from an in-house project.

May I know if could you also please provide the genomics fasta format file used in the construction of UHGGV2plusUHGVmq database?

Thanks in advance!

**Comment by meenachakra** [comment]
Thanks for your interest! Given the interest in custom Phanta DBs, we are working to make a tutorial for making custom Phanta DBs. We hope to complete this within the next few days. We will update you soon (and can also provide the library file at that point)! Please follow up if you have not heard from us by the week of March 5.

**Comment by yipinto** [comment]
Hi Jie, 
Regardless of the tutorial for making Phanta's custom db (which is pretty similar to kraken2 db), you can find the sequences for uhgv here https://github.com/snayfach/UHGV.

**Comment by meenachakra** [comment]
We now have the tutorial ready, please see here - https://github.com/bhattlab/phanta/blob/main/databases.md

Thanks!

**Reply by alienzj** [reply]
Great! Thanks a lot!
